wasm/solana-client/Cargo.toml

```diff
@@ -22,8 +22,8 @@ reqwest = { version = "0.11.9", features = ["json"] }
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
-solana-extra-wasm = { path = "../utils/solana-extra", version = "=1.14.15" }
-solana-sdk = "=1.14.15"
+solana-extra-wasm = { path = "../utils/solana-extra", version = "1.14.16" }
+solana-sdk = "1.14.16"
```

Comment:
Is there a reason why `=` was removed from the versions? Currently this makes it so that the default installation comes with `1.15.2` instead of `1.14.16` if you check `Cargo.lock`

Reply:
Ah, this was unintentional. Updated!